### PR TITLE
MM-4998 Adding LoginIdAttribute to system console.

### PR DESCRIPTION
**Pull request by crspeller**

### CODE REVIEW

**Comment by crspeller** [comment]
@jasonblais Fixed the username help text.
For the command, that's not part of this PR, just the text. I think it's fine as is though. I prefer single word commands if that can be done.

File: components/admin_console/admin_definition.jsx [hunk 1]
```diff
@@ -784,7 +784,7 @@ export default {
                             placeholder: 'admin.ldap.usernameAttrEx',
                             placeholder_default: 'E.g.: "sAMAccountName"',
                             help_text: 'admin.ldap.uernameAttrDesc',
-                            help_text_default: 'The attribute in the AD/LDAP server that will be used to populate the username field in Mattermost. This may be the same as the ID Attribute.',
+                            help_text_default: 'The attribute in the AD/LDAP server that will be used to populate the username field in Mattermost. This may be the same as the LoginIdAttribute.',
```
Comment (jasonblais) [inline components/admin_console/admin_definition.jsx]: Propose "Login ID Attribute", since it's written that way in the UI

**Comment by jasonblais** [comment]
> For the command, that's not part of this PR, just the text. I think it's fine as is though. I prefer single word commands if that can be done.

I'm 0/5, just wondering what is the standard, and why we're changing from what we've done in the past. E.g. `platform user migrate_auth`

**Comment by lieut-data** [comment]
Grepping the `cmd/` directory, it does seem like the convention has traditionally been single word commands:
```
19:     Use:     "system_admin [users]",
78:     Use:   "resetmfa [users]",
95:     Use:     "deleteall",
103:    Use:     "migrate_auth [from_auth] [to_auth] [migration-options]",
25:     Use:   "sampledata",
17:     Use:   "jobserver",
```

I'm 1/5 in favour of underscores, just because they tend to be more readable.

**Comment by crspeller** [comment]
@jasonblais Fixed the "Login ID Attribute" text. 

As for the command, as Jesse points out there is already some inconsistency, so I am just going to leave it as is. This PR doesn't add that command anyway, it's already merged. This is just docs.

**Pull request merged**